eq-author-api/app.js

```diff
@@ -22,7 +22,7 @@ db(process.env.DB_SECRET_ID)
   .then(async conf => {
     let knex = require("knex")(conf);
     logger.info("Running Migrate");
-    await knex.migrate.latest();
+    await knex.migrate.latest({ directory: "migrations" });
```

Comment:
I think `migrations` is the default directory name so probably no need to specify?
https://knexjs.org/#Migrations